report/guilty.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import "regexp"
+import "strings"
+
+func ExtractFiles(log string) []string {
```

Comment:
This should not be part of package public interface.
Make `ExtractGuiltyFile` accept log and return the guilty file, that's what users want.
Tests can use non-exported functions. So make `ExtractFiles` non-exported and call it from `ExtractGuiltyFile` and the test on failure.